src/NHibernate.Test/Async/CfgTest/SettingsFactoryFixture.cs

```diff
@@ -32,6 +32,28 @@ namespace NHibernate.Test.CfgTest
 	public class SettingsFactoryFixtureAsync
 	{
 
+		[Test]
+		public void DefaultServices()
```

Comment:
Won't be generated with the version `0.8.2.10`.

Reply:
Is there anything preventing upgrading to this version in this PR?

Reply:
I upgraded the generator here, if you think that is better to make the upgrade in another PR because of the `CanSetInterceptorAsync` method removed, I will do that.